pom.xml

```diff
@@ -47,6 +47,17 @@
   </properties>
```

Comment:
file has still conflicts:
![image](https://github.com/doubleSlashde/KeepTime/assets/5772511/1149cf9a-67e5-40b3-a4e5-0bcc2c0bf8d3):

Reply:
still has conflicts. as you want to merge it into "feature/macSupport" and you merged develop branch into it - it did not resolve the existing conflicts with the macSupport branch